utils/date.ts

```diff
@@ -63,3 +63,17 @@ export const objectDateToString = (
     : format === 'ISO'
     ? `${year}-${month}-${day}`
     : `${day}-${month}-${year}`;
+
+/** convert all our weird date formats on the /cases api endpoint into iso-compatible strings */
+export const normaliseDateToISO = (str: string): string => {
+  if (/[0-9]{1,2}\/[0-9]{1,2}\/[0-9]{4} [0-9]{1,2}:[0-9:]{1,5}/.test(str)) {
+    const [date, time] = str.split(' ');
+    const [days, month, year] = date.split('/');
+    const [hours, minutes, seconds] = time.split(':');
+    return `${year}-${month}-${days}T${hours.padStart(2, '0')}:${minutes}${
+      seconds ? `:${seconds}` : ''
+    }`;
+  } else {
+    return str;
+  }
+};
```

Comment:
Could something like Moment.js help us here? https://momentjs.com/docs/#/parsing/special-formats/

Reply:
we already have a date library in the app - moment is a very large library so would be best to avoid adding a second one. i think the ideal here is to fix the apis to always expose a standard iso format so we don't need this logic at all!